powerauth-nextstep-client/src/main/java/io/getlime/security/powerauth/lib/nextstep/client/NextStepClient.java

```diff
@@ -2357,11 +2357,13 @@ public ObjectResponse<GetUserAuthenticationListResponse> getUserAuthenticationLi
      * @see #createCredential(CreateCredentialRequest)
      */
     public ObjectResponse<CreateCredentialResponse> createCredential(@NotNull String userId, @NotNull String credentialName, @NotNull CredentialType credentialType,
-                                                                     String username, String credentialValue) throws NextStepClientException {
+                                                                     String username, String credentialValue, CredentialLocation credentialSource, CredentialLocation credentialTarget) throws NextStepClientException {
```

Comment:
From a binary point of view, this is not compatible. Let's depreceate it and prefer `createCredential(@NotNull final CreateCredentialRequest request)`